bin/test_projects_cleanup.rb

```diff
@@ -0,0 +1,78 @@
+example = 'login=rubydev+admin@gooddata.com password=secret server=https://staging2-lcm-prod.intgdc.com ruby delete_projects.rb'
+if ARGV.include?('-h')
+  puts 'Example:'
+  puts example
+  exit 0
+end
+
+require 'gooddata'
+
+def delete_project_by_title(title, projects, days = 14, force = false)
+  dead_line = Time.now - days * 60 * 60 * 24
+  filtered_projects = projects.select do |p|
+    p.title.match(title) && p.created < dead_line
+  end
+  filtered_projects.each do |project|
+    if force
+      puts "Deleting: #{project.pid} - #{project.title} - #{project.created}"
+      project.add && project.add.output_stage && project.add.output_stage.delete
+      project.delete
+    else
+      puts "Would delete: #{project.pid} - #{project.title} - #{project.created}"
+    end
+  end
+  puts "#{filtered_projects.length} projects matching \"#{title}\" #{'would be ' unless force}deleted."
+end
+
+def delete_ads_by_title(title, client, days = 14, force = false)
+  warehouses = client.warehouses
+  return if warehouses.empty?
+  deleted = 0
+  warehouses.each do |warehouse|
+    next unless warehouse.title.match(title)
```

Comment:
info: Refers to 'warehouse' more than self (maybe move it to another class?) (FeatureEnvy) (reek)